internal/pkg/api/handleCheckin.go

```diff
@@ -210,7 +210,12 @@ func (ct *CheckinT) processRequest(zlog zerolog.Logger, w http.ResponseWriter, r
 	if err != nil {
 		return errors.Wrap(err, "subscribe policy monitor")
 	}
-	defer ct.pm.Unsubscribe(sub)
+	defer func() {
+		err := ct.pm.Unsubscribe(sub)
+		if err != nil {
+			zlog.Error().Err(err).Str("policy_id", agent.PolicyId).Msg("unable to unsubscribe")
```

Comment:
[Suggestion]
Add a bit more context to the message, so even who is unfamiliar with the agent or this error will have a more complete understand of the problem when reading the log.

```suggestion
			zlog.Error().Err(err).Str("policy_id", agent.PolicyId).Msg("unable to unsubscribe from policy")
```